tsdb/index/index.go

```diff
@@ -1064,77 +1130,72 @@ func (r *Reader) Close() error {
 }
 
 func (r *Reader) lookupSymbol(o uint32) (string, error) {
```

Comment:
Lot's of repetitions here for label values on series decode, are we sure this is fine overall?

Reply:
That's the cpu vs ram tradeoff this PR is making.